devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/buildtool/gradle/codestart.yml

```diff
@@ -17,11 +17,11 @@ language:
         guide: https://quarkus.io/guides/gradle-tooling
         guide-native: https://quarkus.io/guides/gradle-tooling#building-a-native-executable
         cmd:
-          dev: ./gradlew quarkusDev
-          package: ./gradlew quarkusBuild
-          package-uberjar:  ./gradlew quarkusBuild --uber-jar
-          package-native: ./gradlew build -Dquarkus.package.type=native
-          package-native-container: ./gradlew build -Dquarkus.package.type=native -Dquarkus.native.container-build=true
+          dev: gradle quarkusDev
+          package: gradle quarkusBuild
+          package-uberjar:  gradle quarkusBuild --uber-jar
```

Comment:
Just checked. This flag just sets `-Dquarkus.package.uber-jar=true` but it's not marked for deprecation, so I guess we're good

Reply:
Cool!